libafl_cc/src/clang.rs

```diff
@@ -42,6 +44,9 @@ impl LLVMPasses {
                 .join(format!("cmplog-routines-pass.{}", dll_extension())),
             LLVMPasses::AFLCoverage => PathBuf::from(env!("OUT_DIR"))
                 .join(format!("afl-coverage-pass.{}", dll_extension())),
+            LLVMPasses::Dict2File => {
```

Comment:
Change the name to Autodict, it doesn't seem to write to file

Reply:
Tokens would be more accurate, since it's not `key:value`